CMakeLists.txt

```diff
@@ -32,28 +36,35 @@ ENDIF()
 # Activate this to error on all things deprecated in 5.15
 #add_compile_definitions("QT_DISABLE_DEPRECATED_BEFORE=0x050F00")
 
-# If Qt6, set series number to 1.
-IF(Qt6_FOUND)
-     SET(STELLARIUM_MAJOR 1)
+# two last digits of the current year
+STRING(TIMESTAMP SHORTYEAR "%y")
+MATH(EXPR STELLARIUM_MAJOR_DIGIT "${SHORTYEAR} + 0" OUTPUT_FORMAT DECIMAL)
+# Version of Stellarium has format YY.V.YD, where:
+# -- YY (MAJOR) - two last digits of the year of release
+# -- V  (MINOR) - version of the release (0 is used before first release)
+# -- YD (PATCH) - 0 for releases and the day of the current year for snapshots
```

Comment:
What do you call a patch release of 23.4 released on December 30, 2023, and the next on January 15, 2024? 
This looks like 

23.4.364
23.4.015

Reply:
No, 23.4.364 & 24.0.15

Reply:
The first idea for third number was count the days from latest release, but this is not possible via pure cmake. So, the second idea is just using number of day in the current year and switch middle number to 0 when new series is started in 1 January (we may do not publish any snapshots after December release).

Reply:
Ah, numbers 24.0.* are dev snapshots from January to March! OK.

Reply:
Yes, 24.1.* for dev snapshots from March to June...